### PR TITLE
built-ins/DataView/*: make all indentation consistent (depth & character)

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: test/built-ins/DataView/prototype/getBigInt64/return-value-clean-arraybuffer.js [hunk 1]
```diff
@@ -36,13 +36,13 @@ features: [DataView, ArrayBuffer, BigInt]
...
-assert.sameValue(sample.getBigInt64(1, true), 0n, "sample.getBigInt64(1, true)");
-assert.sameValue(sample.getBigInt64(2, true), 0n, "sample.getBigInt64(2, true)");
-assert.sameValue(sample.getBigInt64(3, true), 0n, "sample.getBigInt64(3, true)");
-assert.sameValue(sample.getBigInt64(4, true), 0n, "sample.getBigInt64(4, true)");
-assert.sameValue(sample.getBigInt64(0, false), 0n, "sample.getBigInt64(0, false)");
-assert.sameValue(sample.getBigInt64(1, false), 0n, "sample.getBigInt64(1, false)");
-assert.sameValue(sample.getBigInt64(2, false), 0n, "sample.getBigInt64(2, false)");
-assert.sameValue(sample.getBigInt64(3, false), 0n, "sample.getBigInt64(3, false)");
-assert.sameValue(sample.getBigInt64(4, false), 0n, "sample.getBigInt64(4, false)");
+assert.sameValue(sample.getBigInt64(0, true), 0 n, "sample.getBigInt64(0, true)");
```
Comment (ljharb) [inline test/built-ins/DataView/prototype/getBigInt64/return-value-clean-arraybuffer.js]: These look like they’d be a syntax error?

Comment (anba) [reply test/built-ins/DataView/prototype/getBigInt64/return-value-clean-arraybuffer.js]: This is all over the place in the BigInt changes and part of the reason I've requested to revert the first PR.

Comment (rwaldron) [reply test/built-ins/DataView/prototype/getBigInt64/return-value-clean-arraybuffer.js]: @ljharb Yes, I thought I had removed these.

**Comment by rwaldron** [comment]
@ljharb comments addressed

**Pull request merged**